deploy.sh

```diff
@@ -75,6 +75,9 @@ if [ -d lcr-report ]; then
     cd lcr-report && git pull && cd -
 else
     git clone https://github.com/liuyq/android-report.git lcr-report
+    if [ -v SECRET_KEY ]; then
```

Comment:
for this SECRET_KEY setting, it still needs the user to setup the environment before run this deploy.sh, otherwise the problem is still there. it would be better to setup a random value for it to make it work by default.

Reply:
I don't know what the `SECRET_KEY` is used for, just know that it's needed.

The user can set it in the environment before running `deploy.sh`. This improves the deployment process so that no further modifications are required.

Reply:
me neither:(
some random value should be OK for the SECRET_KEY.